src/nativeapm/elements/inputs/email.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference path="../../references.ts" />
+
+/**
+ * ProcessOut module/namespace
+ */
+module ProcessOut {
+  /**
+   * ProcessOut Native APM class for handling email input
+   */
+  export class NativeApmEmailInput extends NativeApmTextInput {
+    emailRegex = /^\S+@\S+$/i;
```

Comment:
I took this regex from Mobile SDK documentation. I agree that it's very simple, but since Mobile SDK is using that I would stick with the same to keep it consistent, WDYT?

Regarding type="email" - agree, I will add it!

Reply:
I think keeping it consistent with the Mobile SDK is a good idea, I do agree! Does make me think we should consider updating the Mobile SDK as well though 😅 probably should be a separate action though

Reply:
In mobile SDK the idea was to let backend handle validation properly and use simple regex just to filter the most obvious cases.